app/views/groups/members/index.html.erb

```diff
@@ -24,6 +24,7 @@
               dark:divide-gray-700
             "
           >
+            <%= turbo_frame_tag("member-update-alert") %>
```

Comment:
I'm curious as to how this is working properly (and it seems to be looking at the generated html.  You have this set within the table tag, and it is not a table element.  Rails seems to be adding the alert before the table.  Interesting.

![](https://github.com/phac-nml/irida-next/assets/11295750/b65622ef-af5a-43a7-89fe-1323f6b7cba6)

![](https://github.com/phac-nml/irida-next/assets/11295750/3c9c1897-817e-4a35-8718-01439ded9959)

Reply:
Oh wow good catch it's supposed to be on the outside of the table. Very interesting indeed

Reply:
Updated this in 67b1261203e2417028fdc8854c4454b402ece0ef

Reply:
Thanks